recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "netpbm" %}
+{% set version = "10.86.4" %}
+{% set rev = "r3652" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  svn_url: http://svn.code.sf.net/p/netpbm/code/stable
+  svn_rev: {{ rev }}
+  svn_ignore_externals: True
```

Comment:
Is there a tarball to use? https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
For the stable releases (as included in this package), no. They only make available tarballs for the "super stable" releases. I have nothing against change from stable to super stable, but they are a bit outdated. If it's better, I can change it to super stable tarballs.

Reply:
It is is better to use tarballs for the reasons listed.